core/spec.md

```diff
@@ -4293,11 +4317,8 @@ HTTP/1.1 204 No Content
 
 ### Inlining
 
-The `inline` query parameter MAY be used on requests or responses to indicate
-how nested collections or certain (potentially large) attributes are to be
-handled.
-
-#### Inlining GET Responses
+The `inline` query parameter MAY be used on requests to indicate how nested
```

Comment:
Is this now *only* valid on read requests, not put/post/patch/etc? If so, I think it would be good to clarify that. (Or maybe clarify the difference between inline and nested anyway...)

Reply:
ok clarification added

Reply:
Definitely better, thanks :)